src/rules/no-unresolved.js

```diff
@@ -53,15 +53,17 @@ module.exports = function (context) {
     const modules = call.arguments[0]
     if (modules.type !== 'ArrayExpression') return
 
-    for (let element of modules.elements) {
-      if (element.type !== 'Literal') continue
-      if (typeof element.value !== 'string') continue
-
-      if (element.value === 'require' ||
-          element.value === 'exports') continue // magic modules: http://git.io/vByan
-
-      checkSourceValue(element)
-    }
+    modules.elements.forEach((element) => {
```

Comment:
Ahhh. Realized that returns didn't do the same thing inside the foreach, but didn't click on the fact that they did do the same thing as a continue. I agree with you. Will go back and take advantage of that insight. It twists your code a little less.

Frankly, I prefer your original code style and have used it in many similar cases for more than 30 years despite the CompSci perfectionists always criticizing me for not having a single point of return at the end of my functions. I'm a CompEng. We prefer to do what works. :smile: